lib/fpm/package/rpm.rb

```diff
@@ -301,6 +301,11 @@ def converted_from(origin)
         end
       end.flatten
     end
+   
+    # Erase troublesome release candidate declarations.
+    self.dependencies = self.dependencies.collect do |dep|
+      dep.gsub(/.rc\d*/, '')
```

Comment:
prerelease gems can have any suffix, not just `rcN`... not sure how to solve this.